GPU/GPUTracking/TPCClusterFinder/ClusterAccumulator.cxx

```diff
@@ -19,11 +19,19 @@
 using namespace GPUCA_NAMESPACE::gpu;
 using namespace GPUCA_NAMESPACE::gpu::tpccf;
 
-GPUd() void ClusterAccumulator::toNative(const ChargePos& pos, Charge q, int minSplitNum, tpc::ClusterNative& cn, const GPUTPCGeometry& geo) const
+GPUd() bool ClusterAccumulator::toNative(const ChargePos& pos, Charge q, tpc::ClusterNative& cn, const GPUParam& param) const
 {
-  bool isEdgeCluster = CfUtils::isAtEdge(pos, geo.NPads(pos.row()));
-  bool wasSplitInTime = mSplitInTime >= minSplitNum;
-  bool wasSplitInPad = mSplitInPad >= minSplitNum;
+  cn.qTot = mQtot; // TODO: mQtot is float and qTot is an integer. Can mQtot be non-integral? If yes, we should round
```

Comment:
This is an issue. Originally mQtot was always integral. With the addition of the deconvolution thats no longer the case.